components/openindiana/nwam-manager/Makefile

```diff
@@ -85,3 +87,14 @@ build: $(BUILD_32)
 install: $(INSTALL_32)
 
 download:: $(SOURCE_DIR)/.downloaded
+
+REQUIRED_PACKAGES += gnome/config/gconf
+REQUIRED_PACKAGES += library/desktop/atk
+REQUIRED_PACKAGES += library/desktop/gtk2
+REQUIRED_PACKAGES += library/desktop/libglade
+REQUIRED_PACKAGES += library/glib2
+REQUIRED_PACKAGES += library/gnome/gnome-libs
```

Comment:
I noticed that it was a meta-package, but that's what `gmake REQUIRED_PACKAGES' gave me.  I don't know of any other way to determine the list of dependancies.  Are you suggesting that I use your list instead?  I can certainly do that.